src/data/community-events.json

```diff
@@ -197,6 +197,15 @@
     "startDate": "2023-03-31",
     "endDate": "2023-04-02"
   },
+  {
+    "title": "ETH Miami/ ETH Mobile",
```

Comment:
```suggestion
    "title": "ETH Mobile Miami",
```

Reply:
Updating title to avoid confusion, seems `ETHMobile Miami` is the name of the event according to the site/social media, could you confirm?

Reply:
Thank you so much @nhsz  

It is the first festival/hackathon focused on the mobile experience based on feedback from partners and enterprise of our global Ethereum ecosystem (Metamask, Consensys and WalletConnect) but we will want to continue to be " ETH Miami " in this case because thats how we started and how we consistently got EF Ecosystem Support funding. 

" ETH Miami " will be the best description. Thank you so much.